alts.tex

```diff
@@ -977,7 +977,7 @@ \section{Where do I go from here?}
 (which has plenty of citations, but is more clearly written than any of them).
 
 Dan Boneh also offers excellent courses in cryptography online, for free. They
-can be accessed at the following URL's.
+can be accessed at the following URLs.
```

Comment:
In Canadian English the `'` is correct. But sure, I'll take it.

Reply:
Oh interesting... TIL